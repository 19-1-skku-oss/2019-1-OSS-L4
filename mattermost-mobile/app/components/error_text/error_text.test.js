// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
// See LICENSE.txt for license information.

import React from 'react';
import {shallow} from 'enzyme';
import Preferences from 'mattermost-redux/constants/preferences';

import ErrorText from './error_text.js';

describe('ErrorText', () => {
    const baseProps = {
        textStyle: {
            fontSize: 14,
            marginHorizontal: 15,
        },
        theme: Preferences.THEMES.default,
        error: {
            message: 'Username must begin with a letter and contain between 3 and 22 characters including numbers, lowercase letters, and the symbols',
        },
    };

    test('should match snapshot', () => {
        const wrapper = shallow(
            <ErrorText {...baseProps}/>,
        );

        expect(wrapper.getElement()).toMatchSnapshot();
    });
});
