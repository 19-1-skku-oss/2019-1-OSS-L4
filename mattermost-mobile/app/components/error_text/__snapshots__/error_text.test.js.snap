// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorText should match snapshot 1`] = `
<Text
  style={
    Array [
      Object {
        "color": "red",
        "fontSize": 12,
        "marginBottom": 15,
        "marginTop": 15,
        "textAlign": "left",
      },
      Object {
        "color": "#fd5960",
      },
      Object {
        "fontSize": 14,
        "marginHorizontal": 15,
      },
    ]
  }
>
  Username must begin with a letter and contain between 3 and 22 characters including numbers, lowercase letters, and the symbols
</Text>
`;
